e2e/linodemachine-controller/vpc-integration/chainsaw-test.yaml

```diff
@@ -0,0 +1,130 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: vpc-integration
+spec:
+  steps:
+  - name: step-00
+    try:
+    - assert:
+        file: 00-assert.yaml
+  - name: step-01
+    try:
+    - apply:
+        file: 01-create-cluster.yaml
+    - assert:
+        file: 01-assert.yaml
+  - name: step-02
+    try:
+    - apply:
+        file: 02-create-linodemachine.yaml
+    - assert:
+        file: 02-assert.yaml
+  - name: step-03
+    try:
+    - script:
+        env:
+        - name: TARGET_API
+          value: api.linode.com
+        - name: TARGET_API_VERSION
+          value: v4beta
+        - name: URI
+          value: linode/instances
+        - name: FILTER
+          value: '{"tags":"vpc-machine-test"}'
+        content: |
+          set -e
+          curl -s \
+            -H "Authorization: Bearer $LINODE_TOKEN" \
+            -H "X-Filter: $FILTER" \
+            -H "Content-Type: application/json" \
+            "https://$TARGET_API/$TARGET_API_VERSION/$URI"
+        check:
+          ($error): ~
+          (json_parse($stdout)):
+            results: 1
+    - script:
+        env:
+        - name: TARGET_API
+          value: api.linode.com
+        - name: TARGET_API_VERSION
+          value: v4beta
+        - name: URI
+          value: vpcs
+        - name: FILTER
+          value: '{"label":"machine-test-vpc"}'
+        content: |
+          set -e
+          curl -s \
+            -H "Authorization: Bearer $LINODE_TOKEN" \
+            -H "X-Filter: $FILTER" \
+            -H "Content-Type: application/json" \
+            "https://$TARGET_API/$TARGET_API_VERSION/$URI"
+        check:
+          ($error): ~
+          (json_parse($stdout)):
+            active: true
```

Comment:
This was the only failure in the tests, I might need your help crafting the correct syntax here. The returned JSON for reference is
```
{
  "data": [
    {
      "id": 423512,
      "label": "machine-test-vpc",
      "description": "",
      "region": "us-sea",
      "subnets": [
        {
          "id": 8765312,
          "label": "test-subnet",
          "ipv4": "10.0.0.0/24",
          "ipv6": null,
          "linodes": [
            {
              "id": 123452346,
              "interfaces": [
                {
                  "id": 542346,
                  "active": true
                }
              ]
            }
          ],
          "created": "2024-03-07T14:33:12",
          "updated": "2024-03-07T14:33:12"
        }
      ],
      "created": "2024-03-07T14:33:12",
      "updated": "2024-03-07T14:33:12"
    }
  ],
  "page": 1,
  "pages": 1,
  "results": 1
}

```
I think we want something like this
`data.linodes[0].interfaces[0].active: true` but I'm not sure the exact syntax to get that result

Reply:
this should do it
```yaml
data:
- subnets:
  - linodes:
    - interface:
      - active: true
```

Reply:
Fixed